ReactApp/src/misc/api-fetch-setup.js

```diff
@@ -15,12 +15,12 @@ import { getGBKit } from './store';
  * and preloads some endpoints with mock data for specific components.
  */
 export function initializeApiFetch() {
-	const { siteURL, authToken } = getGBKit();
-	const rootURL = siteURL ? `${siteURL}/wp-json/` : undefined;
+	const { siteAPIURL, authToken, siteID } = getGBKit();
+	const rootURL = siteAPIURL ?? undefined;
 
 	apiFetch.use(apiFetch.createRootURLMiddleware(rootURL));
```

Comment:
I introduced `rootURL` solely to avoid passing a partial URL (e.g., `undefined/wp-json` to the API fetch configuration. With this full URL value provided by the host app, I think we should remove `rootURL` and pass `siteAPIURL` directly to `apiFetch.createRootURLMiddleware`. WDYT?

```suggestion

	apiFetch.use(apiFetch.createRootURLMiddleware(siteAPIURL));
```

Reply:
Updated in https://github.com/wordpress-mobile/GutenbergKit/pull/15/commits/7656e760f3a3ae2fc13162e41ddeff90d4a4f725